lifecycle-operator/controllers/common/helperfunctions_test.go

```diff
@@ -381,3 +384,44 @@ func Test_MergeMaps(t *testing.T) {
 		})
 	}
 }
+
+func Test_resourceRefUIDIndexFunc(t *testing.T) {
+	type args struct {
+		rawObj client.Object
+	}
+	tests := []struct {
+		name string
+		args args
+		want []string
+	}{
+		{
```

Comment:
can we introduce a case when UID == "" ?

Reply:
sure thing